Library/Formula/cegui.rb

```diff
@@ -0,0 +1,54 @@
+class Cegui < Formula
+  desc " provides windowing and widget toolkits for GL and other APIs"
+  homepage "http://cegui.org.uk/"
+  url "https://bitbucket.org/cegui/cegui/get/v0-8-4.tar.gz"
+  version "0.8.5"
+  sha256 "a435a170e3ca6cca4a32ce369e12d803757264dd0d064935d7b9b4f69c8832a8"
+
+  head do
+    url "https://bitbucket.org/cegui/cegui", :using => :hg, :branch => "v0-8"
+  end
+
+  # TODO: Try python option again when CEGUI issue #1114 is resolved
```

Comment:
Unless we can include these now or the next version is imminent enough that we can wait for it in this PR let's scrap the "in future" notations.